cmd/templ/fmtcmd/testdata.txtar

```diff
@@ -22,7 +22,7 @@ templ b() {
 	<div><p>B
 </p></div>
 }
--- a.templ --
+-- b.templ --
```

Comment:
based on the file context I *think* this was a typo and should be "b" so I fixed it here. Let me know if my understanding is in error and I'll revert (or you can, you should have branch edit rights)